src/data_access/User/userDataAccessObject.java

```diff
@@ -19,14 +19,16 @@ public User createUser(String username, String password){
     }
 
     @Override
-    public String storeUser(User user, String PasswordCheck) {
+    public String storeUser(User user, String password, String PasswordCheck) {
         if (!user.verifyPassword(PasswordCheck)) {
             return "Password not same";
         }
         if (user.getUsername().equals("")){
             return "Empty";
         }
-
+        if (password.length() < 5){
+            return "Your password is too small. 🦑";
+        }
```

Comment:
- Good that you fixed the no-password bug. Even better that you added that it must be a length of 5.
- Good that you have error message